<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FuryMainWindow</class>
 <widget class="QMainWindow" name="FuryMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fury Engine :: Car AI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../FuryRecources.qrc">
    <normaloff>:/images/icon</normaloff>:/images/icon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="FuryRenderer" name="openGLWidget">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,0,0,0,0,0,0,0,0">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QTreeView" name="tvSceneObjects"/>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="objectProperties">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1203</width>
             <height>268</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3"/>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1203</width>
             <height>262</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="lbCameraZoom">
              <property name="text">
               <string>Увеличение камеры: 90</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="cameraZoomSlider">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="minimum">
               <number>35</number>
              </property>
              <property name="maximum">
               <number>130</number>
              </property>
              <property name="value">
               <number>90</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="shadowNearLabel">
              <property name="text">
               <string>ShadowNear: 0.1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="shadowNearSlider">
              <property name="value">
               <number>0</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="shadowPlaneLabel">
              <property name="text">
               <string>ShadowPlane: 70</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="shadowPlaneSlider">
              <property name="value">
               <number>60</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="shadowViewSizeLabel">
              <property name="text">
               <string>ShadowViewSize: 25</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="shadowViewSizeSlider">
              <property name="value">
               <number>15</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="shadowCamDistanceLabel">
              <property name="text">
               <string>ShadowCamDistance: 37</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="shadowCamDistanceSlider">
              <property name="value">
               <number>64</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="learnSpeedCheckBox">
          <property name="text">
           <string>Ускоренно</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="debugRenderCheckBox">
          <property name="text">
           <string>Отладочный рендер</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveModelButton">
          <property name="text">
           <string>Сохранить модель</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbMaterialEdit">
          <property name="text">
           <string>Редактор материалов</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbManagers">
          <property name="text">
           <string>Просмотр данных менеджеров</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Нагрузка приложения на систему:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="nothingAction"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="aboutQtAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="quitAction">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="nothingAction">
   <property name="text">
    <string>Ничего не делать</string>
   </property>
  </action>
  <action name="aboutQtAction">
   <property name="text">
    <string>Об qt</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FuryRenderer</class>
   <extends>QOpenGLWidget</extends>
   <header>FuryRenderer.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../FuryRecources.qrc"/>
 </resources>
 <connections/>
</ui>
